backend/src/authentication/authentication.consts.ts

```diff
@@ -0,0 +1,2 @@
+export const saltRounds = 10;
+export const jwt = { secret: 'Remember to make me more secure before prod!', expiry: `7d` };
```

Comment:
This is a potential security bug

Reply:
The goal is to move this into environment variables when preparing the environment files for UAT Deploy (S1T1).

Reply:
Have resolved this in advance to ensure it is enforced later.